etc/kubernetes/manifests/overlays/e2e/kustomization.yaml

```diff
@@ -0,0 +1,86 @@
+resources:
+  - ../dev
+
+commonLabels:
+  e2e.cos.bf2.org/group: "cos"
+
+patchesStrategicMerge:
+  - |-
+    $patch: delete
+    apiVersion: camel.apache.org/v1
+    kind: IntegrationPlatform
+    metadata:
+      name: camel-k
+
+patches:
+  #
+  # sync
+  #
+  - target:
```

Comment:
Can't we change the target to `kind: Deployment` only and use the same for all of them?

Reply:
I think it was not possible last time I checked, I may explore other transformers later on